slides/content/02-kotlin/_index.md

```diff
@@ -1206,10 +1203,9 @@ Kotlin's type system supports generics
 * No higher kinded types (they are in Scala)
 * No type lambdas (they are in Scala)
 * Declaration-site variance (absent in Java)
-* Generic type reification via inlining (not found in Java nor Scala)
```

Comment:
We discuss about the reification mechanism some time ago. Probably is not the same thing, but now the Scala inlining mechanism is similar to the Kotlin one (probably is more powerful and dangerous since metaprogramming concerns).
The example that you show in the slide in Kotlin could be written in Scala as:
https://scastie.scala-lang.org/cric96/v0gO5pKiS9SHCuU4OAs08w/5